x11/windowmaker/Portfile

```
@@ -38,26 +48,20 @@ depends_lib \
 	port:giflib \
 	port:tiff
 
-post-patch {
-	set fl [glob -directory ${worksrcpath} WindowMaker/*menu*]
-	lappend fl ${worksrcpath}/WindowMaker/Defaults/WindowMaker.in \
-	               ${worksrcpath}/WPrefs.app/Menu.c ${worksrcpath}/WPrefs.app/Paths.c
-	foreach i $fl {
-		reinplace "s|/usr/X11R6/share|${prefix}/share|g" $i
-		reinplace "s|/usr/share|${prefix}/share|g" $i
-		reinplace "s|/usr/local/share|${prefix}/share|g" $i
-		reinplace "s|/opt/share|${prefix}/share|g" $i
-		reinplace "s|/opt/kde2/share|${prefix}/share|g" $i
-	}
-}
+configure.env   LINGUAS="bg cs da de el es et fi fr gl hr hu it ja \
+                ko ms nl no pl pt ro ru sk sv tr zh_CN"
 
-configure.env   "LINGUAS=bg cs da de el es et fi fr gl hr hu it ja \
-                ko ms nl no pl pt ro ru sk sv tr zh_CN" \
-                "LIBS=-L${prefix}/lib -lintl -lX11"
 configure.args  --mandir=${prefix}/share/man \
-                --with-nlsdir=${prefix}/share/locale \
-                --with-gnustepdir=${prefix}/GNUstep/Local
+    --with-gnustepdir=${prefix}/GNUstep/Local \
+    --disable-magick 
```

There is trailing whitespace on this line.
```suggestion
    --disable-magick
```